### DISCUSSION TITLE
Notifications position

### DISCUSSION THREAD

**Discussion by olegsvs** [linuxdeepin/developer-center#3058] [Features Request & Ideas | 特性请求 & 头脑风暴]
How to change notifications position from top to bottom?

**Comment by BLumia** [comment]
Currently not supported to change notification position.

**Comment by meiyixiang** [comment]
Regarding the issue of notification, we will comprehensively optimize the notification mechanism in the future

**Comment by meiyixiang** [comment]
You might even change your current notification completely
